modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2040,7 +2265,13 @@ def binary_op(self, op, right_frame, join_type="outer"):
         new_columns = self.columns.join(right_frame.columns, how=join_type)
         return self.__constructor__(new_frame, joined_index, new_columns, None, None)
 
-    def concat(self, axis, others, how, sort):
+    def concat(
+        self,
+        axis: int,
+        others: Union["ModinDataframe", List["ModinDataframe"]],
```

Comment:
why we say `"ModinDataframe"` as string here? we should be able to directly reference the class as this piece is building the descendant of it

Reply:
Shouldn't this be a `"PandasDataframe"`? Or do we expect to be able to join and concat non-identically-backed dataframes (e.g. Omnisci + pandas on Ray)?

Reply:
I guess it should be a `PandasDataframe`, I don't think we expect to join different natures, at least not at the beginning.
Also, I wonder why it's not in the base class now...